packages/dev/core/src/Buffers/bufferUtils.ts

```diff
@@ -117,6 +124,34 @@ export function GetTypeByteLength(type: number): number {
     }
 }
 
+/**
+ * Gets the appropriate TypedArray constructor for the given component type.
+ * @param componentType the component type
+ * @param context an optional context to provide additional information on error
+ * @returns the constructor object
+ */
+export function GetTypedArrayConstructor(componentType: number, context?: string): TypedArrayConstructor {

```

Comment:
I think maybe remove the context and add a try/catch to the calling in the loader to add the context. It's weird for a generic function to have this parameter.